docs/faq.md

```diff
@@ -0,0 +1,29 @@
+# FAQ
+
+* What does MUO stand for?
+
+managed-upgrade-operator
+
+* Does MUO silence alerts?
+
+Yes. All non-critical alerts are silenced during an upgrade. Specific "noisy" critical alerts are also silenced and can be found [here](https://github.com/openshift/managed-cluster-config/blob/ba87886f4d096f5760e907878afe127860318792/deploy/managed-upgrade-operator-config/10-managed-upgrade-operator-configmap.yaml#L12-L20).
```

Comment:
Change link to `https://github.com/openshift/managed-cluster-config/blob/master/deploy/managed-upgrade-operator-config/10-managed-upgrade-operator-configmap.yaml#L12-L20`